providers/fips/fipsprov.c

```diff
@@ -850,6 +855,11 @@ int ERR_clear_last_mark(void)
     return c_clear_last_error_mark(NULL);
 }
 
+void ERR_clear_last_constant_time(int clear)
+{
+    c_clear_last_constant_time(NULL, clear);
+}
+
```

Comment:
This is problematic - if the fips provider is run against older versions of core which should be supported, this call will crash as c_clear_last_constant_time will be NULL.

We need to have some workaround in case c_clear_last_constant_time is NULL even if the workaround won't be constant time.

Maybe @levitte or @mattcaswell could have some ideas on how this could be workarounded. BTW currently the functions that depend on existing err_clear_last_constant_time() in the FIPS provider just manipulate the error stack without calling the core. It might be an option to have this fallback here as well.